GOVERNANCE.md

```diff
@@ -4,51 +4,163 @@ This document defines governance policies for the Kiali project.
+* ability to write high quality code and/or documentation,
+* ability to collaborate with the team,
+* understanding of team policies and processes,
+* understanding of the project's code base, and coding and documentation style.
+
+A new maintainer must be proposed by an existing maintainer by opening a pull request
+modifying the maintainers list to add the new proposed maintainer, or via a [GitHub
+discussion](https://github.com/kiali/kiali/discussions/new) under the Governance category.
+The following information must be provided:
+
+* nominee's GitHub username,
+* an explanation of why the nominee should be a maintainer,
+* a list of links to non-trivial pull requests (top 10) authored by the nominee.
 
 Two other maintainers need to second the nomination. If no one objects in 5 working days (U.S.), the nomination is accepted.  If anyone objects or wants more information, the maintainers discuss and usually come to a consensus (within the 5 working days). If issues can't be resolved, there's a simple majority vote among current maintainers.
 
-## Changes in Maintainership
+## Testers
+
+Kiali testers dedicate time and resources to ensure that the Kiali project is
+delivered with good quality, by actively trying pull requests to find bugs,
+performance issues and any other kind of defect. Testers may also write manual or
+automated tests. The focus is on "System testing" and "Integration testing",
+although it is possible to do simple sanity, smoke and regression testing, or
+simply running existent automated tests if that is enough for a certain code change.
+
+Testers are granted the same privileges as Maintainers, and are also invited to become
+a member of the Kiali organization. This is in good faith of not performing tasks of a
+Maintainer.
+
+Testers do not need to be Maintainers. Also, Maintainers do not need to be Testers. Yet,
+both roles aren't mutually exclusive.
+
+Current list of testers (alphabetically ordered):
+
+* [FilipB](https://github.com/FilipB)
+* [mattmahoneyrh](https://github.com/mattmahoneyrh)
+* [pbajjuri20](https://github.com/pbajjuri20)
+* [prachiyadav](https://github.com/prachiyadav)
+* [skondkar](https://github.com/skondkar)
+
+### Becoming a Tester
+
+To become a Tester you need to:
+
+* actively participate in testing code changes of the Kiali project
+  * testing pull requests for 3 months or more,
+  * find 5 non trivial defects in Kiali,
+  * occassionally, do testing over `master` branches to find broken features
+* ability to collaborate with the team,
+* ability to document testing procedures of Kiali features and to update existing documentation if features change,
+* understanding of team policies and processes.
+
+You can propose yourself to apply for the Tester role, or a Maintainer can do it for you.
+Proposals must be posted by opening a pull request
+modifying the testers list to add the new proposed tester, or via
+a [GitHub discussion](https://github.com/kiali/kiali/discussions/new) under the
```

Comment:
why not settle on a single way of proposing this?

Reply:
My take is that this document exposes how to properly formalize (i.e. it's a process we cannot skip), but I don't think there should strictly be a single way to do add somebody.

I think a new member could be introduced in a meeting and if everybody gives their acknowledgement you already have reached that "simple majority voting" in advance, and we can go directly to a PR to quickly formalize (and be in compliance with governance). However, some people may not be confortable with a PR, so there is the chance of a simple discussion to also formalize.